src/net/sourceforge/kolmafia/maximizer/Evaluator.java

```diff
@@ -1002,10 +1003,11 @@ void enumerateEquipment(int equipScope, int maxPrice, int priceLevel)
     }
 
     int usefulSynergies = 0;
-    Iterator syn = Modifiers.getSynergies();
+    Iterator<Entry<String, Integer>> syn = Modifiers.getSynergies();
```

Comment:
Then if this returned a map, just use forEach here (and other similiar places) too